shared/logutil/logutil.go

```diff
@@ -23,3 +26,19 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// CountdownToGenesis logs the time remains until genesis time
+func CountdownToGenesis(genesisTime time.Time, secondsCount int) {
+	ticker := time.NewTicker(time.Duration(secondsCount) * time.Second)
+
+	for {
+		select {
+		case <-time.NewTimer(genesisTime.Sub(roughtime.Now()) + 1 /* adding one to be able to print the last minute */).C:
```

Comment:
```suggestion
		case <-time.After(genesisTime):
```
This is much simpler

Reply:
It didn't work because it's time, After only take duration and I didn't find an easy way to convert it